### ISSUE TITLE
jsfiddle / codepen sample

### ISSUE THREAD

**Issue by paganaye** [Download/polymer-cdn#2]
It took me sometime to understand how to get polymer working with github raw files.
Here is what I end it up with:
jsfiddle: https://jsfiddle.net/paganaye/h932rkr8/5/
codepen: http://codepen.io/paganaye/pen/QjWezW?editors=100

Is it what you had in mind?

**Comment by Download** [comment]
Yes, except for one small, but important, detail.

You should not link to master files. Instead you should always link to a specific version. For two reasons:

1. When you link to the master, your page will automatically pick up any changes I make, immediately. If you want any kind of chance of it not breaking at random on you, don't link to master.
2. RawGit will throttle you and finally even block you if you send too much traffic to urls pointing to master. Versioned urls get a cache expiration date far into the future and RawGit will never throttle you.

Now this is all assuming you will be *using* polymer. If instead you want to hack on Polymer itself, you might consider using the master urls for testing. But I would just fire up a simple HTTP server on your localhost for that personally.

TL;DR;

Use urls with a version in them, like so:

<script src="https://rawgit.com/thomellis/polymer-cdn/1.0.1/lib/webcomponentsjs/webcomponents-lite.min.js"></script>

If you are intending to go into production with code using Polymer from CDN, I recommend forking this repo (like you have done) so you have full control of what happens with the codebase.

If you are missing something or have suggestions, let me know and I'll do my best to help.

**Comment by paganaye** [comment]
Thanks for your reply Stijn.
I understand what you're saying and in fact I had seen the warning in rawgit.com but I am not really sure how  to get the "non-master" version of the file.
Would you mind to give me the path I should be using for let's say webcomponents-lite.min.js?

**Comment by Download** [comment]
Ah yes, I get what you mean.

In the root of the project, [here](https://github.com/Download/polymer-cdn), you'll see a dropdown that by default is set to 'master':

![Branch selection in GitHub](http://i.imgur.com/DxgwHr6.png "Selecting a branch or tag in GitHub")

Once you expand that you can select tags and choose the tag that you want.
